test/core/test_memcpy_zero_bytes.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+#include <string.h>
+
+int main() {
+  char buf[1024];
+  memcpy(buf, (void*)0x0fffffff, 0);
```

Comment:
should this test case just be rolled into test_memcpy2 or test_memcpy3 to avoid increasing the number of files we have to compile during testing?

Reply:
I guess I don't really see what those test are actually testing.    Perhaps we can refactor all of these into a single test_memcpy with 3 different sub-tests as a followup?  But its a little tricky to see what those tests are really testing.